src/index.ts

```diff
@@ -493,12 +493,16 @@ class Envoy {
 				// - Servicelogger response
 				let host_response;
 
+        // Note: we're caluclating cpu time usage of the current process (zomecall) in microseconds (not seconds)
+        const cpuUsage = process.cpuUsage(baselineCpu)        
+        const cpu = cpuUsage.user + cpuUsage.system
+
```

Comment:
Assuming that there's an `await` between these two, the process could switch to another task while that `await` is pending, and in that time any CPU it was using would count towards this measurement. That seems wrong

Reply:
I agree, but I'm not sure of a cleaner way to uniquely track each call at the envoy level unless we're able to somehow generate a unique process ID for each call.  To be truly correct in our calculation, I think the cpu would need to be generated at the conductor level, which is the planned permanent solution.  As this is just a temporary measure and at a more abstract level, my understanding was that some level of inaccuracy was expected.

Reply:
Okay makes sense :)